src/basilisp/lang/vector.py

```diff
@@ -80,9 +83,17 @@ def to_persistent(self) -> "PersistentVector[T]":
         return PersistentVector(self._inner.persistent())
 
 
+reduce_init_obj = object()
```

Comment:
```suggestion
_reduce_sentinel_obj = object()
```

We typically use `_` prefixes to indicate module-private attributes.

Reply:
Thanks, done.